Containerfile.rpmbuild

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/almalinuxorg/8-base:8
+FROM quay.io/centos/centos:stream9
```

Comment:
This can be problematic as we are not going to detect when we build against a libvirt version which is too new compared to what is in rhel8.
I'll reiterate my suggestion that we stop supporting rhel8 and switch to rhel9 minimum which would avoid this issue.

Reply:
@cfergeau yes, should we wait till RHEL-10 release ?

Reply:
I'd be fine with https://libvirt.org/platforms.html
> The project aims to support the most recent major version at all times. Support for the previous major version will be dropped 2 years after the new major version is released or when the vendor itself drops support, whichever comes first. 

RHEL 9 was released in May 2024 almost 3 years ago, so we could follow the same policy and say we no longer support it. Especially as tools that we need (`oc`) have dropped support for it iirc.
But that's a team decision, not something to decide only here :)